config/index.js

```diff
@@ -40,8 +40,8 @@ const config = {
       parameterLimit: process.env.API_PARAMETER_LIMIT
     },
     session: {
-      header_name: 'authorization',
-      secret: process.env.NODE_API_SESSION_SECRET
+      header_name: 'token',
```

Comment:
As I told you the convention is to use the header name `authorization`, but for this training its okay if you use `token`.